### PR TITLE
Minor updates in meta-tegra

### PR DESCRIPTION
**Pull request by ichergui**

### CODE REVIEW

File: external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend [hunk 1]
```diff
@@ -26,3 +26,8 @@ SRC_URI[opticalflow.sha256sum] = "e300c02e4900741700b2b857965d2589f803390849e1e2
 
 # No stable URI is available for NVIDIAOpticalFlowSDK
 INSANE_SKIP:append:cuda = " src-uri-bad"
+
+# Build paths are currently embedded
+INSANE_SKIP:${PN}-dev += "buildpaths"
+INSANE_SKIP:${PN}-src += "buildpaths"
+INSANE_SKIP:libopencv-core += "buildpaths"
```
Comment (kekiefer) [inline external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend:33]: Are these caused by our append?

Comment (ichergui) [reply external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend:33]: No, maybe I should remove that commit

Comment (ichergui) [reply external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend:33]: Tbh, I was not sure about pushing that 😞

Comment (kekiefer) [reply external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend:33]: Yeah I don't think we need to be in the business of fixing up opencv in this layer

**Review by madisongh — CHANGES_REQUESTED** [review]
Thanks for fixing the helper script.  Let's not put the opencv package QA workaround here, though.

File: external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend (hunk 1 above)
Comment (madisongh) [reply external/openembedded-layer/recipes-support/opencv/opencv_4.6.%.bbappend:33]: Yes, please drop this change.

**Comment by ichergui** [comment]
> Thanks for fixing the helper script. Let's not put the opencv package QA workaround here, though.

I will remove that commit. Should we create `.bbappend` in `meta-tegra-community` ?

**Review by madisongh — APPROVED** [review]

**Comment by madisongh** [comment]
> I will remove that commit. Should we create `.bbappend` in `meta-tegra-community` ?

No.  It should be fixed upstream.

**Pull request merged**